### PR TITLE
WPML: Registring the emails sent to donors for translation

### PR DESCRIPTION
**Pull request by amElnagdy**
## Description
The emails sent to donors cannot be translated using WPML even after registering them in the `wpml-config.xml` file.

## Affects
Only one change needed in PHP `/includes/admin/emails/class-email-notification-util.php` and some settings in `wpml-config.xml` file

## What to test
Add the emails sent to donors, for example, Donations -> Settings -> Emails -> Donor Emails and try to translate is using WPML.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: includes/admin/emails/class-email-notification-util.php [hunk 1]
```diff
@@ -285,6 +285,12 @@ public static function get_value( Give_Email_Notification $email, $option_name,
 			? str_replace( '_give_', '', $option_name )
 			: $option_name );
 		$option_value       = give_get_option( $global_option_name, $default );
+		
+		//Emails sent to donors cannot be translated using WPML.
```
Comment (DevinWalker) [inline includes/admin/emails/class-email-notification-util.php:289]: Please update comment to:

`// Ensure that emails sent to donors can be translated using WPML`

File: includes/admin/emails/class-email-notification-util.php [hunk 2]
```diff
@@ -285,6 +285,12 @@ public static function get_value( Give_Email_Notification $email, $option_name,
...
+		//Regiserting only the emails sent to the donors.
```
Comment (DevinWalker) [inline includes/admin/emails/class-email-notification-util.php:290]: Fix typo in "Registering"

File: includes/admin/emails/class-email-notification-util.php [hunk 3]
```diff
@@ -285,6 +285,12 @@ public static function get_value( Give_Email_Notification $email, $option_name,
...
+		if (defined('ICL_SITEPRESS_VERSION') && !is_array($option_value)) {
```
Comment (DevinWalker) [inline includes/admin/emails/class-email-notification-util.php:291]: @ravinderk how come this is not being auto-formatted? Notice the incorrect spacing.

Comment (ravinderk) [reply includes/admin/emails/class-email-notification-util.php:291]: @DevinWalker Maybe @amElnagdy did not set up the development environment correctly or GitHub hook did not install for `husky`. We can automate this process on Github by using GitHub action: https://github.com/rtCamp/action-phpcs-code-review

File: includes/admin/emails/class-email-notification-util.php (hunk 1 above)
Comment (amElnagdy) [reply includes/admin/emails/class-email-notification-util.php]: Done

**Pull request merged**